pyproject.toml

```diff
@@ -235,20 +237,23 @@ warn_unused_configs = true
 warn_unused_ignores = true
 
 [[tool.mypy.overrides]]
-disable_error_code = "attr-defined"
-module = ["tests.*.test_sqlalchemy", "tests.*.test_sqlalchemy"]
-
-[[tool.mypy.overrides]]
-disable_error_code = ["arg-type", "misc", "valid-type", "var-annotated"]
-module = "tests.unit.test_contrib.test_sqlalchemy.test_dto"
+module = ["tests.examples.*", "tests.docker_service_fixtures"]
```

Comment:
Why are these two modules being ignored?

Reply:
They were always ignored by the pre-commit config. When I did the pyright version of this last week there were a stack more issues to resolve, and so picking my battles I chose to leave these two ignored. I've left them ignored in this PR so we are consistent with the pyright config.

If I remove:

![image](https://github.com/litestar-org/litestar/assets/20659309/ac2ed9c2-b5f5-4e8d-895b-88473ed7fedd)

Will be better to remove the excludes from both pyright and mypy in the same PR and fix for both I think. I'll open up an issue for it.